appdatabase/migrations/sql/1722404163_add_status_proxy_urls_to_networks.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE networks ADD COLUMN proxy_rpc_url VARCHAR NOT NULL DEFAULT "";
+ALTER TABLE networks ADD COLUMN proxy_fallback_rpc_url VARCHAR NOT NULL DEFAULT "";
```

Comment:
So we kinda both save the `proxy_rpc_url` to database, yet have dependency on production/test build. 
What's the logic here? When are developers expected to use `test` endpoint?